zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestCluster.java

```diff
@@ -439,4 +439,9 @@ private Stream<TestGateway<?>> allGateways() {
         .filter(TestGateway.class::isInstance)
         .map(node -> (TestGateway<?>) node);
   }
+
+  public void setPartitionCount(final int count) {
```

Comment:
:wrench: I wouldn't add this helper yet, since it's only used in a single place. It seems like a very, very specific use case.